exchanges/bitstamp/bitstamp.go

```diff
@@ -43,7 +43,14 @@ const (
 	bitstampAPITransferFromMain   = "transfer-from-main"
 	bitstampAPIReturnType         = "string"
 	bitstampAPITradingPairsInfo   = "trading-pairs-info"
-	bitstampOHLC                  = "ohlc"
+	bitstampAPIWSAuthToken        = "websockets_token"
+	bitstampAPIWSTrades           = "live_trades"
+	bitstampAPIWSOrders           = "live_orders"
+	bitstampAPIWSOrderbook        = "order_book"
+	bitstampAPIWSMyOrders         = "my_orders"
+	bitstampAPIWSMyTrades         = "my_trades"
+
+	bitstampOHLC = "ohlc"
```

Comment:
Seems like this guy could rejoin his friends

Reply:
If he behaves.

But if I find him smoking crack behind the bike shed again then that's the final warning!